src/koaApi.coffee

```diff
@@ -103,6 +103,7 @@ exports.setupApp = (done) ->
   app.use route.post '/keystore/key', keystore.setServerKey
   app.use route.post '/keystore/ca/cert', keystore.addTrustedCert
   app.use route.get '/keystore/validity', keystore.verifyServerKeys
+  app.use route.post '/keystore/passphrase', keystore.setServerPassphrase
```

Comment:
Do we need a separate API request for just setting the passphrase? I see this is happening in the serServerCert call to. Maybe we can just do it there?